app/main/views/agreements.py

```diff
@@ -16,6 +17,76 @@
 from ...supplier_utils import validate_agreement_details_data
 
 
+@main.route('/agreements', methods=['POST'])
```

Comment:
Is it worth adding a comment that this route isn't currently used, and is not yet supported by the API Client?